### DISCUSSION TITLE
which solution corresponds to Tutorial: Get started with EF Core in an ASP.NET MVC web app

### DISCUSSION THREAD

**Discussion by talln6e** [dotnet/AspNetCore.Docs#23079] [Q&A] [answered]
Which solution corresponds to the source code for this web page?
https://docs.microsoft.com/en-us/aspnet/core/data/ef-mvc/intro?view=aspnetcore-5.0

I can't find it in the zip file folders.
Thank you!

**Comment by Rick-Anderson** [comment, marked as answer]
https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-mvc/intro/samples/5cu

**Reply by sodiqjonovich** [reply]
that's useful
